### DISCUSSION TITLE
How to clear screen when entering room?

### DISCUSSION THREAD

**Discussion by VladDarkel** [evennia/evennia#2603] [Evennia Questions] [answered]
How to clear screen (erase all previous text) when entering room? I couldn't find it in the documentation.

**Comment by ghost** [comment, marked as answer]
This is something that should be handled at the client level since there is no consistent way of achieving this with any degree of success due to the different ways people can connect to the game e.g. telnet or web, compatibility issues with various clients, etc.

For telnet connections you could try sending the screen clear escape sequence ('\e[2J') but not all clients support it (many do not). There is no equivalent for web client connections.

**Reply by VladDarkel** [reply]
Thanks for the answer
